official-allow_functionality.json

```diff
@@ -229,8 +229,43 @@
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.s3-us-west-2.amazonaws.com"}},
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.cloudflare.com"}},
 		{"o":{"h":"*.hansje.net"},"d":{"h":"*.cargocollective.com"}},
-		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}}
+		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}},
+		{"o":{"h":"*.eurobuch.com"},"d":{"h":"*.lehmanns.de"}},
+		{"o":{"h":"*.emp.de"},"d":{"h":"*.acfrg.com"}},
+		{"o":{"h":"*.microsoft.com"},"d":{"h":"*.s-msft.com"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.trackersimulator.org"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.eviltracker.net"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.do-not-tracker.org"}},
+		{"o":{"h":"*.shortfil.ms"},"d":{"h":"*.eay.cc"}},
```

Comment:
Concerning `eay.cc`: sry. Shall I delete it or would you like to? 

Concerning `vimeo.com` and `vimeocdn.com`: well, here at my IT I have generally whitelisted some domains (just like `vimeo.com` and `vimeocdn.com`). I'm going to open a new issue to explain and discuss on that.